eventing/samples/github-source/README.md

```diff
@@ -0,0 +1,160 @@
+# GitHub Source example
+
+GitHub Source example shows how to wire GitHub events for consumption
+by a Knative Service.
+
+## Deployment Steps
+
+### Prerequisites
+
+You will need:
+
+1. An internet-accessible Kubernetes cluster with Knative Serving
+  installed. Follow the [installation
+  instructions](https://github.com/knative/docs/blob/master/install/README.md)
+  if you need to create one.
+  - Ensure Knative Serving is [configured with a domain
```

Comment:
I think you need some extra indent level to get markdown to recognize that these are sub-bullets (shows up as `</ol><ul>` in the HTML right now).